setup.py

```diff
@@ -45,13 +45,21 @@
     'redis>=2.10.0',
 ]
 
+invenio_search_version = '1.0.0b2'
```

Comment:
`1.0.0b3` when https://github.com/inveniosoftware/invenio-search/pull/123 is merged